examples/compiled/point_angle_windvector.vg.json

```diff
@@ -87,7 +87,6 @@
   "legends": [
     {
       "size": "size",
-      "gradientLength": {"signal": "clamp(height, 64, 200)"},
```

Comment:
Do you know why the PNG image generated for this spec changed?

Reply:
Because the png generation isn't stable. I've seen this in a few places. 

Just ignore it when the SVG doesn't change either.

Reply:
It would be good to figure out how we can minimize the noise @kanitw